### ISSUE TITLE
Pixels shift and adjusts when dragging the timer node

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#212]
Discovered during https://github.com/phetsims/wave-interference/issues/205

It looks like this:

![adjusting](https://user-images.githubusercontent.com/679486/49663392-bdc21e80-fa0b-11e8-9b8b-638354a2e98f.gif)

@jonathanolson any idea what is causing this?

**Comment by samreid** [comment]
I asked on slack #dev-public:

> Sam Reid [10:35 AM]
Has anyone seen this problem before, where the pixels shift and adjust while something is moving?
> https://github.com/phetsims/wave-interference/issues/212

**Comment by KatieWoe** [comment]
It shows up in firefox quite a bit.  Give me a minute to find it.

**Comment by samreid** [comment]
Sam Reid [10:35 AM]
Has anyone seen this problem before, where the pixels shift and adjust while something is moving?

https://github.com/phetsims/wave-interference/issues/212

Kathryn Woessner [10:38 AM]
https://github.com/phetsims/wave-interference/issues/212#issuecomment-445307846
Mostly text, but it seems to be a firefox issue.  Is that where you saw it?

Jesse Greenberg [10:39 AM]
Yes, I encountered that fairly recently and Node option preventFit helped https://github.com/phetsims/energy-skate-park/issues/39#issuecomment-439551944

Steele Dalton [10:39 AM]
also preventFit in https://github.com/phetsims/states-of-matter/issues/176

Sam Reid [10:39 AM]
I saw it in Chrome

Steele Dalton [10:39 AM]
yes sam this happens often

**Comment by jonathanolson** [comment]
It definitely looks like moving the timer to the right of the panels is shifting blocks:

![fitted-block-bounds](https://user-images.githubusercontent.com/480878/49747472-bc3d6400-fc60-11e8-8ddd-2f4a5330e96b.PNG)

If this is a common problem, potentially I could turn SVG block fitting off on certain platforms.

I can only seem to reproduce this in Firefox. Does it happen in other browsers? (And the Firefox text quality seems to be unrelated to fitting, but more about motion).

**Comment by KatieWoe** [comment]
@jonathanolson @samreid I looked at Win 10 and Mac 10.14 on Wave Interference master.  I wasn't able to reproduce on firefox on either.  What OS were both of you using?

**Comment by KatieWoe** [comment]
From Slack:
>Jonathan Olson [10:12 AM]
>Interesting. I tested on Win10 / Firefox 63
>updating firefox
>interesting. now it's not "wiggling" during the animation. it only has a "higher quality" resting version, and a "lower quality" version while something is in motion

**Comment by samreid** [comment]
My video in the top comment was recorded on Mac/Chrome

**Comment by KatieWoe** [comment]
Sorry for the confusion.  I thought this was just a firefox problem for some reason.  I didn't see it on Win 10 chrome and I just went to test on Mac 10.14 chrome and still don't see it.  Is it possible something changed in the common code?

**Comment by jonathanolson** [comment]
Unlikely to have been a common code change.

**Comment by samreid** [comment]
Do we publish sims with this symptom?

**Comment by samreid** [comment]
If I put `preventFit: true` on the WaveMeterNode, then the problem goes away for it and for the TimerNode.

**Comment by samreid** [comment]
That commit addresses the problem on my machine.  @jonathanolson am I barking up the right tree?

**Comment by jonathanolson** [comment]
> That commit addresses the problem on my machine. @jonathanolson am I barking up the right tree?

Yes, that would prevent the shifting/resizing of the underlying block.

@ariel-phet since this has now popped up in multiple places, I'm curious if it's worth some time to see when/where fitted blocks should be used. It reduces memory usage and can increase performance (on graphics-bound devices like iPad 3), but if it's causing issues like this then maybe it could be selectively enabled on certain devices?
